validator/rpc/auth_token.go

```diff
@@ -49,18 +41,18 @@ func CreateAuthToken(walletDirPath, validatorWebAddr string) error {
 // user via stdout and the validator client should then attempt to open the default
 // browser. The web interface authenticates by looking for this token in the query parameters
 // of the URL. This token is then used as the bearer token for jwt auth.
-func (s *Server) initializeAuthToken(walletDir string) (string, error) {
-	authTokenFile := filepath.Join(walletDir, AuthTokenFileName)
-	exists, err := file.Exists(authTokenFile, file.Regular)
+func (s *Server) initializeAuthToken() error {
+	if s.authTokenPath == "" {
+		return errors.New("auth token path is empty")
+	}
+	exists, err := file.Exists(s.authTokenPath, file.Regular)
 	if err != nil {
-		return "", errors.Wrapf(err, "could not check if file exists: %s", authTokenFile)
+		return errors.Wrapf(err, "could not check if file exists: %s", s.authTokenPath)
```

Comment:
```suggestion
		return errors.Wrapf(err, "could not check if file %s exists", s.authTokenPath)
```
Colon separates errors